### PR TITLE
Adding more block times

### PR DESCRIPTION
**Pull request by stepanLav**

### CODE REVIEW

File: chains/v13/chains_dev.json [hunk 1]
```diff
@@ -117,7 +117,8 @@
         ],
         "additional": {
             "themeColor": "#E6007A",
-            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/polkadot-and-kusama"
+            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/polkadot-and-kusama",
+            "defaultBlockTime": 6000
```
Comment (stepanLav) [inline chains/v13/chains_dev.json:121]: polkadot

File: chains/v13/chains_dev.json [hunk 2]
```diff
@@ -241,7 +242,8 @@
         ],
         "additional": {
             "themeColor": "#1F78FF",
-            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/polkadot-and-kusama"
+            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/polkadot-and-kusama",
+            "defaultBlockTime": 6000
```
Comment (stepanLav) [inline chains/v13/chains_dev.json:246]: kusama

File: chains/v13/chains_dev.json [hunk 3]
```diff
@@ -329,7 +331,8 @@
...
```
Comment (stepanLav) [inline chains/v13/chains_dev.json:335]: westend

File: chains/v13/chains_dev.json [hunk 4]
```diff
@@ -1274,7 +1277,8 @@
         ],
         "additional": {
             "themeColor": "#E4A228",
-            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains"
+            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains",
+            "defaultBlockTime": 12000
```
Comment (stepanLav) [inline chains/v13/chains_dev.json:1281]: moonriver

File: chains/v13/chains_dev.json [hunk 5]
```diff
@@ -1767,7 +1771,8 @@
         ],
         "additional": {
             "themeColor": "#1F78FF",
-            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains"
+            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains",
+            "defaultBlockTime": 12000
```
Comment (stepanLav) [inline chains/v13/chains_dev.json:1775]: bifrost kusama

File: chains/v13/chains_dev.json [hunk 6]
```diff
@@ -2725,7 +2730,8 @@
         ],
         "additional": {
             "themeColor": "#D632C1",
-            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains"
+            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains",
+            "defaultBlockTime": 12000
```
Comment (stepanLav) [inline chains/v13/chains_dev.json:2734]: calamari

File: chains/v13/chains_dev.json [hunk 7]
```diff
@@ -3462,7 +3468,8 @@
         ],
         "additional": {
             "themeColor": "#10B6B1",
-            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains"
+            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains",
+            "defaultBlockTime": 12000
```
Comment (stepanLav) [inline chains/v13/chains_dev.json:3472]: moonbeam

File: chains/v13/chains_dev.json [hunk 8]
```diff
@@ -5653,7 +5660,8 @@
         ],
         "additional": {
             "themeColor": "#14B1D3",
-            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains"
+            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains",
+            "defaultBlockTime": 12000
```
Comment (stepanLav) [inline chains/v13/chains_dev.json:5664]: turing

File: chains/v13/chains_dev.json [hunk 9]
```diff
@@ -5902,7 +5910,8 @@
         ],
         "additional": {
             "themeColor": "#1F78FF",
-            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains"
+            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains",
+            "defaultBlockTime": 6000
```
Comment (stepanLav) [inline chains/v13/chains_dev.json:5914]: polkadex

File: chains/v13/chains_dev.json [hunk 10]
```diff
@@ -6883,7 +6892,8 @@
         "addressPrefix": 42,
         "additional": {
             "themeColor": "#EB2B6C",
-            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains"
+            "stakingWiki": "https://docs.novawallet.io/nova-wallet-wiki/staking/parachains-and-solochains",
+            "defaultBlockTime": 6000
```
Comment (stepanLav) [inline chains/v13/chains_dev.json:6896]: ternoa

**Review by ERussel — APPROVED** [review]

**Pull request merged**